vm/regexp_test.go

```diff
@@ -0,0 +1,61 @@
+package vm
+
+import (
+	"testing"
+)
+
+func TestRegexpClassSuperclass(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected string
+	}{
+		{`Regexp.class.name`, "Class"},
+		{`Regexp.superclass.name`, "Object"},
+		{`Regexp.ancestors.to_s`, "[Regexp, Object]"},
+	}
+
+	for i, tt := range tests {
+		v := initTestVM()
+		evaluated := v.testEval(t, tt.input, getFilename())
+		checkExpected(t, i, evaluated, tt.expected)
+		v.checkCFP(t, i, 0)
+		v.checkSP(t, i, 1)
+	}
+}
+
+func TestRegexpClassCreation(t *testing.T) {
```

Comment:
Are you planning to uncomment those test cases? Otherwise I think this test function is unnecessary.

Reply:
Uncommented